src/Error/BaseErrorHandler.php

```diff
@@ -75,7 +80,7 @@ public function register()
         set_error_handler([$this, 'handleError'], $level);
         set_exception_handler([$this, 'wrapAndHandleException']);
         register_shutdown_function(function () {
-            if (PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg') {
+            if ((PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg') && $this->_handled) {
```

Comment:
What happens if we remove the skip entirely?

Reply:
The cli output doubles somehow as the (non?)buffered fatal error output is printed and then the display part again.
Note: Could xdebug also be playing a role here? In the duplication?